jekyll/_cci2/api-developers-guide.md

```diff
@@ -0,0 +1,533 @@
+      "node_index": 0,
+      "url": "https://53936-48750547-gh.circle-artifacts.com/0/circleci-docs/assets/img/docs/walkthrough6.png"
+    },
+    {
+      "path": "circleci-docs/assets/img/docs/walkthrough7.png",
+      "node_index": 0,
+      "url": "https://53936-48750547-gh.circle-artifacts.com/0/circleci-docs/assets/img/docs/walkthrough7.png"
+    },
+    {
+      "path": "circleci-docs/assets/img/docs/walkthrough8.png",
+      "node_index": 0,
+      "url": "https://53936-48750547-gh.circle-artifacts.com/0/circleci-docs/assets/img/docs/walkthrough8.png"
+    },
+    ```
+
+4. Next, you may extend this API call to download the artifacts. Navigate to the location you would like to download the artifacts to, and run the following command, remembering to substitute your own values in the request:
+
+     ```sh
+    curl -X GET https://circleci.com/api/v2/project/{VCS}/{org_name}/{repo_name}/{job_number}/artifacts \
+    --header 'Content-Type: application/json' \
+    --header 'Accept: application/json' \
+    --header "Circle-Token: $CIRCLECI_TOKEN" \
+    | grep -o 'https://[^"]*' \
+    | wget -v -i -
+    ```
+
+    **Note:** `grep` is used to locate all the URLs for downloading the job artifacts, while `wget` is used to perform the download.
+
+# Reference
+
+## API Endpoints
+
+The CircleCI v2 API, and its associated endpoints allow you to make HTTP calls to designated endpoints developed in the underlying CircleCI API architecture. These endpoints provide programmatic access to CircleCI services, including pipelines, workflows, and jobs. 
+
+Before working with the API and making calls, you should first have an understanding of the various API requests you can make to API v2 endpoints. The sections below describe these API requests.
+
+### User Endpoints
+
+| Endpoint               | Description                       
+|------------------------|-------------------------------------------------------
+| GET /user/{id}         | This endpoint enables you to return information about a specific user by making a GET request and passing the `id` parameter in the request |
+| GET /me/collaborations | This endoint enables you to return a list of organizations that the user is a member of, or is a collaborator by making a GET request and passing the `collaborations` parameter in the request. |
+{: class="table table-striped"}
+
+### Pipeline API Requests
+
+| Endpoint                                  | Description                                                         | 
+|-------------------------------------------|---------------------------------------------------------------------|
+| GET /pipeline/{pipeline-id}               | Returns an individual pipeline.                                     |
+| GET /pipeline/{pipeline-id}/config        | Returns the configuration of a specific pipeline.                   |
+| GET /pipeline/{pipeline-id}/workflow      | Returns a list of workflows for a specific pipeline.                |
```

Comment:
@thomasr8 I think `/project/{project_slug}` needs adding to the start of these to be consistent

Reply:
Also - from what I can tell there is a pipeline numbers, and a pipeline ID. I think these endpoints should use {pipeline-number}? Not sure what the difference is though!

Reply:
Fixed